src/realm/sync/instruction_replication.hpp

```diff
@@ -198,7 +198,7 @@ class TempShortCircuitReplication {
     bool m_was_short_circuited;
 };
 
-constexpr bool SYNC_SUPPORTS_NESTED_COLLECTIONS = false;
+constexpr bool SYNC_SUPPORTS_NESTED_COLLECTIONS = true;
```

Comment:
this is just for testing...  I will be reverted to false once the code is ready.